packages/libs/error-stack/tests/ui/macro_invalid_args.stderr

```diff
@@ -95,7 +95,7 @@ error[E0308]: mismatched types
    |             arguments to this function are incorrect
    |
 note: associated function defined here
-  --> $RUST/core/src/convert/mod.rs
```

Comment:
I did notice this when testing locally through search. What is this `convert` module? It isn't renamed in the PR which is why I didn't bring it up. Is this change valid?